docs/filldb.md

```diff
@@ -0,0 +1,26 @@
+# Fill Database
+
+## Requirements
```

Comment:
I am not sure why one would need to/want to set the skip api check to false? Could you explain or remove this section if it is not necessary? thx,